libs/datamodel/connectors/dml/src/relation_info.rs

```diff
@@ -33,23 +38,58 @@ impl RelationInfo {
             fields: Vec::new(),
             references: Vec::new(),
             name: String::new(),
-            on_delete: OnDeleteStrategy::None,
+            on_delete: None,
+            on_update: None,
+            legacy_referential_actions: false,
         }
     }
+
+    /// Set referential action legacy mode, skipping the validation errors on
+    /// automatically set actions.
+    pub fn legacy_referential_actions(&mut self) {
+        self.legacy_referential_actions = true;
+    }
 }
 
 /// Describes what happens when related nodes are deleted.
+#[repr(u8)]
+#[bitflags]
```

Comment:
I find this weird — a given relation has a single referential action, so it should be a regular enum, right?

Reply:
Or is that impl used to denote the set of available referential actions for a connector? I'll continue reading.

Reply:
Your guess was right. We can set multiple of them in the connector.